src/bb2.py

```diff
@@ -2,13 +2,90 @@
 Blue Button 2.0 SDK Class
 
 """
+import json
+import os
+import pathlib
+import yaml
+
+ROOT_DIR = os.path.abspath(os.curdir) + "/"
```

Comment:
Just curious - if I import this package, and then run my python server NOT from the top level... will this still work and find my top level config?

Reply:
@nbragdon I was searching around to see what is typically used for packages and found this as one of the simpler ways being used. Not sure if it is the best.

However, I can see this not working, like in the case you mention. We should let the SDK user configure this path, in case this does not work for their setup. I'll add this shortly to the PR.

Also thinking of renaming this to "BASE_DIR".

Reply:
If there is no easy way to do it, I think this is fine. We'll just make sure to include it in the readme. Just wanted to make sure 👍